src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
Well we should do whatever nuget does in that case, you're right. but in this case, this is first run, so project path is not relevant and CWD seems viable.